galaxy_importer/collection.py

```diff
@@ -173,12 +176,56 @@ def _load_collection_manifest(self):
         if not os.path.exists(manifest_file):
             raise exc.ManifestNotFound('No manifest found in collection')
 
+        default_logger.debug('manifest_file: %s', manifest_file)
+
         with open(manifest_file, 'r') as f:
             try:
                 data = schema.CollectionArtifactManifest.parse(f.read())
             except ValueError as e:
                 raise exc.ManifestValidationError(str(e))
             self.metadata = data.collection_info
+            default_logger.debug('data.file_manifest_file: %s', data.file_manifest_file)
+            self.file_manifest_file = data.file_manifest_file
```

Comment:
can we separate the analysis of `FILES.json` and `MANIFEST.json`?

Reply:
Probably. I kind of grabbed a reference to data.file_manifest_file here because currently there isn't a shared reference to the data from CollectionArtifactManifest() except for the 'collection_info' subset.

Could just return 'data' here  (or set self.artifact_manifest_data = data)

Reply:
Or do you mean, validate  CollectionArtifactManifest().collection_info first, and then CollectionArtifactManager().file_manifest_file  ?